### DISCUSSION TITLE
Displaying content on another page

### DISCUSSION THREAD

**Discussion by lukas9812** [QuestPDF/QuestPDF#1068] [Q&A] [answered]
Hello all,
currently I'm struggling with displaying additional content on another page.
Imagine I have custom structure where I draw custom columns every 5 pixels by SkCanvas.DrawLine() and when I'm at the end of content canvas, the content overflow it's size. Ideal situation would be to break page and continue on next page.
I've already tried to use **.BreakPage()**, but this approach do not work for me. Could you please help ? 
![image](https://github.com/user-attachments/assets/500ee58f-2d2c-4ca5-81e4-e15b9b0264fb)

I'm using helper to generate canvas via: https://www.questpdf.com/concepts/skia-sharp-integration.html#integration-layer
Many thanks for help.

**Comment by MarcinZiabek** [comment, marked as answer]
I suspect that the shared custom graphics are dynamically generated. I suggest writing a custom logic that detects how many pages are needed and then uses a DynamicComponent to create each page with its corresponding content.

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
